src/helpers/dataFilter.tsx

```diff
@@ -18,5 +18,16 @@ export const getAllData = async (lat: number, long: number) => {
   return data;
 }
 
+export const getHomeData = async (lat: number, long: number) => {
+	const data: any = {}
+	const weather = await getWeatherData(lat, long)
+	const aq = await getAirQualityData(lat, long)
+	data.temp = +((weather.current.temp - 273.15) * 1.8 + 32).toFixed(0)
```

Comment:
Great Idea doing the temp conversion on the data retrieval. Really clever both of you.